mkdocs/docs/user_guide.md

```diff
@@ -2150,6 +2150,60 @@ There are no time series decoders at the moment (WIP), so time series cannot be
+Audio Features
+--------------
+
+### Audio Features Preprocessing
+
+Ludwig supports reads in audio files using Python's library [SoundFile](https://pypi.org/project/SoundFile/) therefore supporting WAV, FLAC, OGG and MAT files.
+
+-  `in_memory` (default `true`): defines whether image dataset will reside in memory during the training process or will be dynamically fetched from disk (useful for large datasets). In the latter case a training batch of input images will be fetched from disk each training iteration. At the moment only `in_memory` = true is supported.
+- `audio_file_length_limit_in_s`:  (default 5.0) float value that defines the maximum limit of the audio file in seconds. All files longer than this limit are cut off. All files shorter than this limit are padded with `padding_value`
+- `padding_value`: (default 0): float value that is used for padding. 
+- `norm`: (default `null`) the normalization method that can be used for the input data. Supported methods: `null` (data is not normalized), `per_file` (z-norm is applied on a “per file” level)
+- `audio_feature`: (default `{ type: raw }`) dictionary that takes as input the audio feature `type` as well as additional parameters if `type != raw`. The following parameters can/should be defined in the dictionary:
+	- `type` (default `raw`): defines the type of audio features to be used. Supported types at the moment are `raw`, `stft`, `stft_phase`, `group_delay`. For more detail, check [Audio Input Features and Encoders](#audio-input-features-and-encoders).
+	- `window_length_in_s`: defines the window length used for the short time Fourier transformation (only needed if `type != raw`).
+	- `window_shift_in_s`: defines the window shift used for the short time Fourier transformation (also called hop_length) (only needed if `type != raw`).
+	- `num_fft_points`: (default `window_length_in_s * sample_rate` of audio file) defines the number of fft points used for the short time Fourier transformation. If `num_fft_points > window_length_in_s * sample_rate`, then the signal is zero-padded at the end. `num_fft_points` has to be `>= window_length_in_s * sample_rate` (only needed if `type != raw`).
+	- `window_type`: (default `hamming`): defines the type window the signal is weighted before the short time Fourier transformation. All windows provided by [scipy’s window function](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.get_window.html) can be used (only needed if `type != raw`).
+  
+Example of a preprocessing specification (assuming the audio files have a sample rate of 16000):
+```yaml
+name: audio_path
+type: audio
+preprocessing:
+  audio_file_length_limit_in_s: 7.5
+  audio_feature:
+    type: stft
+    window_length_in_s: 0.04
+    window_shift_in_s: 0.02
+    num_fft_points: 800
+    window_type: boxcar
+```
+
+### Audio Input Features and Encoders
+
+Audio files are transformed into one of the following types according to `type` in `audio_feature` in `preprocessing`.
```

Comment:
this sentence and the following list can be moved inside the preprocessing section.